client/post-editor/editor-status-label/index.jsx

```diff
@@ -54,20 +57,12 @@ var StatusLabel = React.createClass( {
 	},
 
 	render: function() {
-		var statusClass = 'post-editor-status-label';
+		let statusClass = classNames( 'editor-status-label', {
+			'is-placeholder': ! this.props.post
+		} );
```

Comment:
This seems a bit odd - it implies the following as a possible configuration (pseudocode)...

`<EditorStatusLabelPlaceholder is-placeholder={ false } />`

I think the `is-placeholder` class should always be applied from within the `Placeholder` component, rather than passed as a prop.